src/bf_dpu_update.py

```diff
@@ -634,6 +657,13 @@ def is_fw_file_for_atf_uefi(self):
         return True
 
 
+    def is_fw_file_for_conf(self):
+        if not self.is_fw_file_for_atf_uefi():
+            return False
+        # TODO: To ditinguish config image from atf_uefi image
```

Comment:
#1. std way, using bfcfg -b XX.bfb, parser the output cfg file:
SYS_ENABLE_OEM_MFG_CONFIG=TRUE
BMC_SOL_ENABLE=yes
BMC_WEBUI_ENABLE=yes
..

#2. simple way but not common
strings BD-config-image-4.9.0.13354-1.0.0.bfb | grep  -i "toutiao"

Reply:
I will choose to use the simple way#2.
Since:
1: bfcfg is not everywhere.
2: config image is only for toutiao. So it's OK for now. I think